api/http_api.md

```diff
@@ -0,0 +1,758 @@
+                "update_time": 1726647649340
+            }
+        ],
+        "total": 2
+    },
+}
+```
+
+- `"error_code"`: `integer`  
+  `0`: The operation succeeds.
+
+  
+The error response includes a JSON object like the following:
+
+```shell
+{
+    "code": 3016,
+    "message": "Can't connect database"
+}
+```
+
+## Get the information of a file of a dataset
+
+**GET** `/api/v1/dataset/{dataset_id}/info`
+
+Get the information of a file of a dataset
+
+### Request
+
+- Method: GET
+- URL: `/api/v1/dataset/{dataset_id}/info`
+- Headers:
+  - `content-Type: application/json`
+  - 'Authorization: Bearer {YOUR_ACCESS_TOKEN}'
+
+#### Request example
+
+```shell
+curl --request GET \
+     --url http://{address}//api/v1/dataset/{dataset_id}/info \
+     --header 'Content-Type: application/json' \
+     --header 'Authorization: Bearer {YOUR_ACCESS_TOKEN}'
+     --data-binary '{
+         "document_id": "4fb0cd625f9311efba4a0242ac120006"
+     }'
+```
+
+#### Request parameters
+
+- `dataset_id`: (*PATH parameter*)
+    The dataset id
+- `"document_id"`: (*Body parameter*)
+   The document id of the file.
+
+### Response
+
+The successful response includes a JSON object like the following:
+
+```shell
+{
+    "code": 0,
+    "data": {
+        "chunk_count": 0,
+        "create_date": "Wed, 18 Sep 2024 06:40:58 GMT",
+        "create_time": 1726641658660,
+        "created_by": "134408906b6811efbcd20242ac120005",
+        "id": "f6b170ac758811efa0660242ac120004",
+        "knowledgebase_id": "779333c0758611ef910f0242ac120004",
+        "location": "story.txt",
+        "name": "story.txt",
+        "parser_config": {
+            "chunk_token_count": 128,
+            "delimiter": "\n!?。；！？",
+            "layout_recognize": true,
+            "task_page_size": 12
+        },
+        "parser_method": "naive",
+        "process_begin_at": null,
+        "process_duation": 0.0,
+        "progress": 0.0,
+        "progress_msg": "",
+        "run": "0",
+        "size": 0,
+        "source_type": "local",
+        "status": "1",
+        "thumbnail": null,
+        "token_count": 0,
+        "type": "doc",
+        "update_date": "Wed, 18 Sep 2024 06:40:58 GMT",
+        "update_time": 1726641658660
+    },
+}
+```
+  
+The error response includes a JSON object like the following:
+
+```shell
+{
+    "code": 3016,
+    "message": "Can't connect database"
+}
+```
+
+
+## Update a file in dataset
+
+**PUT** `/api/v1/dataset/{dataset_id}/documents`
+
+Update a file in a dataset
+
+### Request
+
+- Method: PUT
+- URL: `/api/v1/dataset/{dataset_id}/documents`
+- Headers:
+  - `content-Type: application/json`
+  - 'Authorization: Bearer {YOUR_ACCESS_TOKEN}'
+
+#### Request example
+
+```shell
+curl --request PUT \
+     --url http://{address}//api/v1/dataset/{dataset_id}/info \
+     --header 'Content-Type: application/json' \
+     --header 'Authorization: Bearer {YOUR_ACCESS_TOKEN}'
+     --raw '{
```

Comment:
It's better to move `document_id` to url

Reply:
Fixed